packages/storybook/src/semantic-ui/AudioPlayer.stories.js

```diff
@@ -27,3 +27,11 @@ export const Error = () => (
     src='/path/to/audio.mp3'
   />
 );
+
+export const ErrorLongURL = () => (
+  <AudioPlayer
+    onClose={action('close')}
+    open
+    src='http://localhost:3000/rails/active_storage/blobs/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBZ0lEIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--05fcfd55941c73ff139611bcc194a158f08fe5a5/sample_640x426.mp3'
```

Comment:
I realize this is just a storybook component, but the localhost URL for the sample video  looks odd -- won't that fail when viewing the storybook?

Reply:
Ahh yes, I should have explained more in the description: That's actually what I'm testing in this story, a URL that fails to load the image/content. The path of a resource with a long URL was running off the modal, so I updated the CSS to break the text properly.

## Before
![Screen Shot 2022-08-23 at 10 51 31 AM](https://user-images.githubusercontent.com/20641961/186190214-f8adc739-3c7b-4f9e-8332-ca9bf59b6e18.png)

## After
![Screen Shot 2022-08-23 at 10 48 51 AM](https://user-images.githubusercontent.com/20641961/186189762-7cb45104-f846-4c41-9ecf-231b94785eb0.png)